From: Punit Agrawal <punit@debian.org>
Date: Fri, 6 Jul 2018 23:24:25 +0100
Subject: Don't run the elvish test during build

Support for elvish in direnv adds a dependency on being able to
download and build the elvish shell. This doesn't fit with distro
package builds where elvish is packaged separately.

Also, the elvish tests depend on set-env which doesn't seem to be part
of elvish. Drop running the tests until this dependency is properly
managed.
---
 Makefile | 2 --
 1 file changed, 2 deletions(-)

diff --git a/Makefile b/Makefile
index 95fec1f..d5c8155 100644
--- a/Makefile
+++ b/Makefile
@@ -50,8 +50,6 @@ man: $(ROFFS)
 test: build
 	go test
 	./test/direnv-test.sh
-	go get github.com/elves/elvish
-	elvish ./test/direnv-test.elv
 
 install: all
 	install -d $(DESTDIR)$(PREFIX)/bin
